### PR TITLE
Cleaned Up Readme

### PR DESCRIPTION
**Pull request by ghost**
- Cleaned up readme
- Verified that this works currently

### CODE REVIEW

File: content/newappmesssage.txt [hunk 1]
```diff
@@ -1 +1 @@
-Try Our Fresh New App|Preview the upcoming much improved Emby Roku app.  A beautifully redesigned interface that allows you to view your Emby content on your Roku like never before.  Try it now by installing the private preview channel from http://emby.media/emby-for-roku.html
+Try Our Fresh New App|Preview the upcoming much improved Emby Roku app.  A beautifully redesigned interface that allows you to view your Jellyfin content on your Roku like never before.
```
Comment (JustAMan) [inline content/newappmesssage.txt:1]:
```suggestion
Try Our Fresh New App|Preview the upcoming Jellyfin Roku app.  A beautifully redesigned interface that allows you to view your Jellyfin content on your Roku like never before.
```

**Review by JustAMan — CHANGES_REQUESTED** [review]
Huh, why upload a `release.zip` to the repo instead of github releases? I don't think it should be here.

Plus please coordinate with #10.

File: content/newappmesssage.txt (hunk 1 above)
Comment (anthonylavado) [reply content/newappmesssage.txt:1]: I agree with the above suggestions. If you need help creating a release to GitHub's tracker, let us know and we can arrange it :-)

**Comment by JustAMan** [comment]
@winters-brown I've merged #10, please use it as base and update accordingly.

File: content/newappmesssage.txt (hunk 1 above)
Comment (ghost) [reply content/newappmesssage.txt:1]: Ah, sorry about the release file. Its a artifact I forgot to delete before pushing. Also im not entirely familiar with githubs release tracker? do you mean this 

screenshot from 2019-02-15 06-24-59

also i will get ontop coordinating with #10 as soon as possible. I have been buried in school work lately and haven't checked github in a while.

Comment (JustAMan) [reply content/newappmesssage.txt:1]: Yes, we meant this tracker. So if there's a need to have a release zip up there (if that helps anyone) contact us (e.g. in matrix channels), and we can arrange it.

BTW, I've merged #10, so take a look there. Also note there's another PR #13, you probably need to incorporate it as well.

**Comment by JustAMan** [comment]
@winters-brown please rebase your PR on top of what's currently in master.

**Comment by ghost** [comment]
Im going to close this because right now i don't have much time to work on this because of school and extra curricular, sorry. Soon i will have some more time to get back on working on this project. Just not currently.

**Pull request closed without merging**